src/sidebar/components/AutocompleteList.tsx

```diff
@@ -4,30 +4,45 @@ import { useMemo } from 'preact/hooks';
 
 import MenuArrow from './MenuArrow';
 
-/**
- * @template T
- * @param {T} item
- */
-const defaultListFormatter = item => item;
+const defaultListFormatter = <Item,>(item: Item) => item;
```

Comment:
You'd better bet your biscuits I had to look this syntax up. I realize I'm at the rear of the caboose of our little TypeScript-learning train, but in case this is new to anyone else: the comma here pacifies the TS compiler because we're in TSX land. As I understand it, `extends` would also work, e.g. `<Item extends unknown>`.

Reply:
I didn't know about this either. Yet another option would have been to convert this to a named function `function defaultListFormatter<Item>(...) { ... }`.